projects/feedback/feedback_experiment.py

```diff
@@ -289,7 +390,14 @@ def getL4PredictiveCells(self):
     """
     Returns the predictive cells in L4.
     """
-    return [set(column._tm.getPredictiveCells()) for column in self.L4Columns]
+    # ApicalTMRegion uses "getPredictedCells", while ExtendedTMRegion uses "getPredictiveCells".
+    #return [set(column._tm.getPredictiveCells()) for column in self.L4Columns]
+    if self.L4RegionType == "py.ApicalTMRegion":
+        return [set(column._tm.getPredictedCells()) for column in self.L4Columns]
```

Comment:
In this PR you're using a lot of 4-space tabs. Can you convert these all to 2-spaces-per-tab? It's a little messy seeing a mix of 4 and 2.